magazinerepack stringtable fix #5390

shukari opened the pull request:
stringtable optimization

- fixes #5390

commy2 commented on addons/magazinerepack/stringtable.xml:
```diff
@@ -132,7 +132,7 @@
             <English>%1 Full and %2 Partial</English>
             <French>%1 plein(s) et %2 partiel(s)</French>
-            <German>%1 volle Magazine und 1 Magazin mit %2 Partronen</German>
+            <German>%1 volle Magazine und %2 angefangenes Magazine</German>
```
Maybe `Magazin(e)` so it works with "1"? Same for `volle(s)` und `angefangene(s)`.

shukari replied:
In 90% of the cases you have >1 full and 1 partial... so it is optimised for the 90%... (e) looks so ugly or?

shukari commented:
This issue is so small but i find no really good translation... here some optinions:

With ():
%1 volle(s) Magazin(e) und %2 angefangene(s) Magazin(e)
%1 volle(s) und %2 angefangene(s) Magazin(e)
%2 angefangene(s) und %1 volle(s) Magazin(e)

With the prediction for the most cases (full >1 and partial = 1):
%2 angefangenes und %1 volle Magazine -> favorite, because "%1 volle Magazine" is the important info
%1 volle und %2 angefangenes Magazine

shukari commented:
I changed the translation to @commy2 suggestion and changed the message for successful repacking to full mags and x rounds left.
I hope that fits for all.

jonpas commented:
@commy2 ?

commy2 commented on addons/magazinerepack/stringtable.xml:
```diff
@@ -144,6 +96,6 @@
             <Chinesesimp>%1个满的与%2个部分的</Chinesesimp>
             <Chinese>%1個滿的與%2個部分的</Chinese>
-        </Key>
+          </Key>
```
spaces

shukari replied:
fixed
